Astronomy/Utilities/NetworkDataLoader.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  NetworkDataLoader.swift
+//  Astronomy
+//
+//  Created by Craig Swanson on 2/12/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+protocol NetworkDataLoader {
+    func loadData(from request: URLRequest, completion: @escaping (Data?, Error?) -> Void)
+    
+    func loadData(from url: URL, completion: @escaping ( Data?, Error?) -> Void)
```

Comment:
Try implementing a Typealias for the completion parameter. That way you wont repeat the same code in multiple places.